core-bundle/contao/classes/BackendUser.php

```diff
@@ -132,6 +132,9 @@ public function __get($strKey)
 
 			case 'fop':
 				return \is_array($this->arrData['fop']) ? $this->arrData['fop'] : ($this->arrData['fop'] ? array($this->arrData['fop']) : false);
+
+			case 'alexf':
+				return (array) ($this->arrData['alexf'] ?? null);
```

Comment:
This would return an empty array for an empty string value.

```suggestion
				return empty($this->arrData['alexf']) ? array(): $this->arrData['alexf'];
```

Reply:
wouldn't ensure an array though 🤔  maybe this is better?
```suggestion
				return \is_array($this->arrData[$strKey] ?? null) ? $this->arrData[$strKey] : array();
```